scripts/generate_dump

```diff
@@ -1154,6 +1173,9 @@ disable_logrotate() {
 #  None
 ###############################################################################
 enable_logrotate() {
+    if [ ! -f "/etc/cron.d/logrotate" ]; then
```

Comment:
why do we need this change, logrotate is present on sup as well.

Reply:
Currently, when we login to platform Sup and LC to execute "show techsupport", these two functions (disable_logroate() and enable_logrote()) report issue as below -- No such file or directory.  Also, I verified that "logrotate" is not present in /etc/cron.d/ directory.  File is not present in Pilot Image 202205.72 image as well as in the 202405

disable_logrotate() reports below
```
sed: can't read /etc/cron.d/logrotate: No such file or directory
handle_error $? $LINENO
ERR: RC:-2 observed on line 1289
```

enable_logrotate() retports below:
```
sed: can't read /etc/cron.d/logrotate: No such file or directory
handle_error $? $LINENO
ERR: RC:-2 observed on line 1329
```
This is why I add the check